### PR TITLE
Adding OneDriveLoopSharingCapability

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/issues/197

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2012,6 +2013,33 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -OneDriveLoopSharingCapability
+
+When sharing a whiteboard in a Teams meeting, Whiteboard creates a sharing link that’s accessible by anyone within the organization and automatically shares the whiteboard with any in-tenant users in the meeting.
+
+There’s an additional capability for temporary collaboration by external and shared device accounts during a meeting. This allows these users to temporarily view and collaborate on whiteboards when they’re shared in a Teams meeting, similar to PowerPoint Live sharing.
+
+If you have the external sharing for ODB allowed, no further action is required. If you have external sharing disabled, you can leave it disabled but you must enable this new setting. For more information, see [Enable Microsoft Whiteboard for your organization](https://support.microsoft.com/en-us/office/enable-microsoft-whiteboard-for-your-organization-1caaa2e2-5c18-4bdf-b878-2d98f1da4b24).
```
Comment (tseward) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2022]: Please remove 'en-us'

**Review by tseward — APPROVED** [review]

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md (hunk 1 above)
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2022]:
```suggestion
If you have the external sharing for ODB allowed, no further action is required. If you have external sharing disabled, you can leave it disabled but you must enable this new setting. For more information, see [Enable Microsoft Whiteboard for your organization](https://support.microsoft.com/office/enable-microsoft-whiteboard-for-your-organization-1caaa2e2-5c18-4bdf-b878-2d98f1da4b24).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Review by dariomws — APPROVED** [review]

**Pull request merged**